examples/charts/file/area.py

```diff
@@ -1,6 +1,9 @@
 from collections import OrderedDict
 
-from bokeh._legacy_charts import Area, show, output_file
+from bokeh.charts import Area, show, vplot, output_file, defaults
+
+defaults.width = 400
+defaults.height = 400
 
 # create some example data
 xyvalues = OrderedDict(
```

Comment:
We can do this work in a separate PR but if so please make an issue to explicitly do a pass over the examples to make them all consistent, concise, non-redundant, etc.